docs/faq.md

```diff
@@ -1251,6 +1251,20 @@ Then, for different backends:
     --parallel=1
     ```
 
+#### How to skip compatibility architecture checks when using unsupported model architectures with custom backend versions?
```

Comment:
How can I skip unsupported model architecture checks within a higher version of vLLM?